spaces/S000175/README.md

```diff
@@ -0,0 +1,19 @@
+---
+uid: S000175
+slug: radial-plane
+name: Radial plane
+aliases:
+- Core topology on $\mathbb R^2$
+refs:
+  - mathse: 2250999
+    name: "Is the core topology on R2 a group topology?"
+  - mr: MR0069388
+    name: Some topological properites on convex sets (Klee 1955)
+  - mr: MR0044014
+    name: Convex sets in linear spaces (Klee 1951)
+  - doi: 10.2307/2315929
+    name: Problem 5468 of The American Mathematical Monthly
+---
+
+Sets $U$ are open provided for every point $x$ and line passing through $x$,
+an open subinterval of the line is a subset of $U$.
```

Comment:
Mention that $x$ is in $U$ and the open subinterval contains $x$.  Something like "... for every point $x\in U$ and line passing through $x$, the point $x$ is contained in an open subinterval of the line that is a subset of $U$" for example.